profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/perimeters.tf

```diff
@@ -12,6 +12,40 @@ resource "google_folder" "folder" {
   display_name = each.key
 }
 
+# Perimeter folder admins are users with direct administrative access to the
+# folder. For example, in All of Us, the primary service account uses these
+# roles to perform administrative tasks on workspaces, such as associating the
+# free tier billing account, renaming files, or checking Stackdriver egress
+# metrics.
+#
+# This is useful when administrative accounts are not owners of the workspaces
+# in their perimeter, or when they own too many workspaces and normal Google
+# Group access via Sam no longer functions:
+# https://docs.google.com/document/d/1-nc7hwqhM-pdJlsR-41u7zgy43kEKf8tptGhTuwbgUk/edit
+resource "google_folder_iam_policy" "folder-admin-policy" {
+  for_each = var.folder_admins
+
+  folder      = google_folder.folder[each.key].name
+  policy_data = data.google_iam_policy.perimeter-policy[each.key].policy_data
+}
+
+data "google_iam_policy" "perimeter-policy" {
```

Comment:
@dvoet are there existing IAM policies on the folders that we wouldn't want to manage through Terraform - if not, are there other bindings I missed here? For example, presumably the Rawls SA has some kind of permission on the folder to create projects here. Or is this all granted at the GCP org level?

If we don't want to fully manage it here, that's fine, I can switch to https://www.terraform.io/docs/providers/google/r/google_folder_iam_binding.html which should leave existing bindings unaffected (as long as they are different roles) - or the smallest granularity https://www.terraform.io/docs/providers/google/r/google_folder_iam_member.html to avoid all risk of conflict

Reply:
I updated this - assuming there are other policies on here, since I know at minimum the Terraform SA is on the folder policy, and we certainly don't want to remove that.

I used the binding approach, which I believe will overwrite any other members for these particular roles. I'm hoping this doesn't cause any issues. The member-based approach is too verbose.

@gmalkov run the following to import state before applying to check:
```
terraform import 'google_folder_iam_binding.admin-bucket-writer["terra_dev_aou_test"]' "737976594150 organizations/400176686919/roles/terraBucketWriter"
terraform import 'google_folder_iam_binding.admin-billing-manager["terra_dev_aou_test"]' "737976594150 roles/billing.projectManager"
terraform import 'google_folder_iam_binding.admin-monitoring-viewer["terra_dev_aou_test"]' "737976594150 roles/monitoring.viewer"
```